hasher-matcher-actioner/webapp/src/pages/ContentDetails.tsx

```diff
@@ -67,73 +67,78 @@ export default function ContentDetailsSummary(): JSX.Element {
   }, []);
 
   return (
-    <FixedWidthCenterAlignedLayout title="Summary">
-      <Row>
-        <Col className="mb-4">
-          <ReturnTo />
-        </Col>
-      </Row>
-      <Row
-        style={{
-          minHeight: '450px',
-        }}>
-        <Col md={6}>
-          <h3>Content Details</h3>
-          <Table>
-            <tbody>
-              <tr>
-                <td>Content ID:</td>
-                <td>{id}</td>
-              </tr>
-              <tr>
-                <td>Last Submitted:</td>
-                <td>
-                  {contentDetails && contentDetails.updated_at
-                    ? formatTimestamp(contentDetails.updated_at)
-                    : 'Unknown'}
-                </td>
-              </tr>
-              <tr>
-                <td>Additional Fields:</td>
-                <td>
-                  {contentDetails && contentDetails.additional_fields
-                    ? contentDetails.additional_fields.join(', ')
-                    : 'No additional fields provided'}
-                </td>
-              </tr>
-              <tr>
-                <td>Content Hash:</td>
-                <CopyableHashField
-                  text={
-                    hashDetails
-                      ? hashDetails.content_hash ?? 'Not found'
-                      : 'loading...'
-                  }
-                />
-              </tr>
-              <tr>
-                <td>Last Hashed on:</td>
-                <td>
-                  {hashDetails
-                    ? formatTimestamp(hashDetails.updated_at)
-                    : 'loading...'}
-                </td>
-              </tr>
-            </tbody>
-          </Table>
-          <ActionHistoryTable contentKey={id} />
-        </Col>
-        <Col className="pt-4" md={6}>
-          {img && contentDetails && contentDetails.content_type ? (
-            <ContentPreview
-              contentId={id}
-              contentType={getContentTypeForString(contentDetails.content_type)}
-              url={img}
-            />
-          ) : null}
-        </Col>
-      </Row>
-      <ContentMatchTable contentKey={id} />
-    </FixedWidthCenterAlignedLayout>
+    <FullWidthLocalScrollingLeftAlignedLayout title="Summary">
```

Comment:
blocking: (kinda): This layout is meant for use where the children implement their own scrolling by using the `overflow` css property. See how [Matches.tsx](https://github.com/facebook/ThreatExchange/blob/main/hasher-matcher-actioner/webapp/src/pages/Matches.tsx#L257) does it. 

In your present impl, if the page becomes longer than a screenful, users will not be able to scroll vertically at all.

Reply:
You could create a `FullWidthLeftAlignedLayout` component that copies from `FixedWidthCenterAlignedLayout` but uses container-fluid.

Reply:
Done